src/teleop/gui/yarn-error.log

```diff
@@ -0,0 +1,4363 @@
+Arguments: 
```

Comment:
Can we please not push this to git. Unless there's some reason I'm missing but seems like a massive file we don't need

Reply:
Agree. Is there a git ignore? If not, we should probably add one

Reply:
Yeah there is, should be a simple addition

Reply:
Oh I missed this when I made the PR, I had though all the yarn files were set up in gitignore oops